apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/instdata/InstdataClient.java

```diff
@@ -80,10 +83,20 @@ public void gjenopprett(RsDollyUtvidetBestilling bestilling, DollyPerson dollyPe
     @Override
     public void release(List<String> identer) {
 
-        List<String> environments = instdataConsumer.getMiljoer();
-        environments.forEach(environment ->
-                identer.forEach(ident -> instdataConsumer.deleteInstdata(ident, environment))
-        );
+        try {
+            instdataConsumer.deleteInstdata(identer)
+                    .subscribe(response -> log.info("Slettet antall {} identer fra Instdata",
```

Comment:
"Slettet antall {} identer ..." høres litt feil ut for meg. Vil foreslå å bruke "Slettet {} identer... " i stedet her og de andre stedene "Slettet antall" brukes :)